third_party/docfx-doclet-143274/src/main/java/com/microsoft/build/YmlFilesBuilder.java

```diff
@@ -19,16 +22,18 @@
 import jdk.javadoc.doclet.DocletEnvironment;
 
 public class YmlFilesBuilder {
-  private DocletEnvironment environment;
-  private String outputPath;
-  private ElementUtil elementUtil;
-  private PackageLookup packageLookup;
-  private String projectName;
-  private boolean disableChangelog;
-  private ProjectBuilder projectBuilder;
-  private PackageBuilder packageBuilder;
-  private ClassBuilder classBuilder;
-  private ReferenceBuilder referenceBuilder;
+  private static final String OLDER_AND_PRERELEASE = "Older and prerelease packages";

```

Comment:
(out of ignorance) - what's the purpose of making these `final`?

Reply:
Discussed offline; making fields `final` is preferred if possible in general as it helps the compiler optimize and prevents unwanted behavior (assuming that having the field be unchanged is indeed the desired behavior).